.github/workflows/cockpit-lib-update.yml

```diff
@@ -0,0 +1,27 @@
+name: cockpit-lib-update
+on:
+  schedule:
+    - cron: '0 2 * * 4'
+  # can be run manually on https://github.com/cockpit-project/starter-kit/actions
+  workflow_dispatch:
+jobs:
+  cockpit-lib-update:
+    runs-on: ubuntu-20.04
```

Comment:
I'd rather this would be `ubuntu-latest`, which is 22.04 now. Let's please not introduce workflows on an already outdated Ubuntu version.